includes/Core/Util/Tracking.php

```diff
@@ -194,14 +209,18 @@ private function inline_js_admin_data( $data ) {
 	 * Register tracking settings and allow access from Rest API.
 	 *
 	 * @since 1.0.0
+	 * @since n.e.x.t Registers a meta field instead of setting.
 	 */
 	private function register_settings() {
+		global $wpdb;
 		$args = array(
 			'type'         => 'boolean',
 			'description'  => __( 'Allowing tracking of anonymous usage stats.', 'google-site-kit' ),
 			'default'      => false,
-			'show_in_rest' => true,
+			'single'       => true,
+			'show_in_rest' => current_user_can( Permissions::SETUP ),
 		);
-		register_setting( 'google-site-kit', self::TRACKING_OPTIN_KEY, $args );
+		// Need to include the blog prefix as this is a user option.
+		register_meta( 'user', $wpdb->get_blog_prefix() . self::TRACKING_OPTIN_KEY, $args );
```

Comment:
This is right, but looks kind of strange. I wonder whether we should include a generic "core-like" `googlesitekit_register_user_option()` function that wraps this. Potentially in a new `wp-functions.php` file or so that is included in `load.php`. Just an idea, what do you think? What I'd want to make sure though is that we only use that file for such closely core-related things that are not offered by it, not for Site Kit utilities.

Reply:
I'm not opposed to adding some functions although I think we should keep them under the `Google\Site_Kit` namespace.

Reply:
Good point. Anyway, let's leave this using `register_meta()` for now, but what we actually need to do here is only add `$wpdb->get_blog_prefix()` if not in network mode.

Reply:
Ok will do. To your point about creating an abstraction here, this may be a good reason to create a `User_Option` class, similar to how we have a `Setting` class now? This is probably best handled in a new issue but seems like it would be a good way to go about solving this.

Reply:
I've added a new issue for tackling this larger work in the future here https://github.com/google/site-kit-wp/issues/1029